cfme/tests/automate/custom_button/test_container_objects.py

```diff
@@ -105,3 +109,66 @@ def test_custom_button_display(request, display, setup_obj, button_group):
         custom_button_group = Dropdown(view, group.hover)
         assert custom_button_group.is_displayed
         assert custom_button_group.has_item(button.text)
+
+
+@pytest.mark.uncollectif(
+    lambda appliance, button_group: not bool([obj for obj in OBJ_TYPE_59 if obj in button_group])
+    and appliance.version < "5.10"
+)
+def test_custom_button_dialog(appliance, dialog, request, setup_obj, button_group):
+    """ Test custom button with dialog and InspectMe method
+
+    Prerequisites:
+        * Appliance
+        * Simple TextInput service dialog
+
+    Steps:
+        * Create custom button group with the Object type
+        * Create a custom button with service dialog
+        * Navigate to object Details page
+        * Check for button group and button
+        * Select/execute button from group dropdown for selected entities
+        * Fill dialog and submit
+        * Check for the proper flash message related to button execution
+        * Check request in automation log
+    """
+
+    group, obj_type = button_group
+
+    # Note: No need to set display_for dialog only work with Single entity
+    button = group.buttons.create(
+        text=fauxfactory.gen_alphanumeric(),
+        hover=fauxfactory.gen_alphanumeric(),
+        dialog=dialog,
+        system="Request",
+        request="InspectMe",
+    )
+    request.addfinalizer(button.delete_if_exists)
+
+    view = navigate_to(setup_obj, "Details")
+    custom_button_group = Dropdown(view, group.hover)
+    assert custom_button_group.has_item(button.text)
+    custom_button_group.item_select(button.text)
+
+    dialog_view = view.browser.create_view(TextInputDialogView)
+    dialog_view.wait_displayed()
+    assert dialog_view.service_name.fill("Custom Button Execute")
+
+    # Clear the automation log
+    assert appliance.ssh_client.run_command('echo -n "" > /var/www/miq/vmdb/log/automation.log')
```

Comment:
This can be done with `touch /var/www/.../automation.log`

Reply:
are you sure about this? can we clean file with `touch`?
```
/tmp ❯❯❯ echo "hi Nikhil" > temp.log
/tmp ❯❯❯ cat temp.log 
hi Nikhil
/tmp ❯❯❯ touch temp.log 
/tmp ❯❯❯ cat temp.log 
hi Nikhil
```